onnxruntime/core/framework/allocation_planner.h

```diff
@@ -78,33 +79,29 @@ class SequentialPlannerContext : public ISequentialPlannerContext {
 // but we can't assume any execution order between sequences, unless there is a data dependency.
 class IGraphPartitioner {
  public:
-  // DeviceBasedPartition is the default, who partitions a graph based off device information.
-  // i.e., given a graph which has CPU EP nodes, Cuda EP nodes and TRT EP nodes,
-  // it will be partitioned as two sequences, one is for CPU EP nodes, another is for TRT and Cuda nodes.
-  // We will add more optimized partitioner later.
   enum GraphPartitioningStrategy {
     DeviceBasedPartition = 0,
     Unknown,
   };
+  // DeviceBasedPartition is the default, who partitions a graph based off device information.
+  // i.e., given a graph which has CPU EP nodes, Cuda EP nodes and TRT EP nodes,
+  // it will be partitioned as two sequences, one is for CPU EP nodes, another is for TRT and Cuda nodes.
```

Comment:
shouldn't be 3 sequences for this case? 1 for cpu, 1 for TRT, 1 for Cuda?

Reply:
TRT and Cuda EP nodes should all fall into Cuda device category.